web-registry/src/components/molecules/BatchTotalsGrid.tsx

```diff
@@ -1,14 +1,11 @@
 import React from 'react';
 import { Grid, SxProps, Theme } from '@mui/material';
+import { quantityFormatNumberOptions } from 'config/decimals';
```

Comment:
just using the formatting config that's already used throughout the whole app:
```
export const QUANTITY_MIN_DECIMALS = 0;
export const QUANTITY_MAX_DECIMALS = 6;

export const quantityFormatNumberOptions = {
  minimumFractionDigits: QUANTITY_MIN_DECIMALS,
  maximumFractionDigits: QUANTITY_MAX_DECIMALS,
};
```

cc/ @clevinson in the case of C56-002-20140701-20141231-001, I don't think that makes sense to display decimals when the value is already an integer, so I'm still using 0 for `minimumFractionDigits`